site/content/docs/5.3/content/typography.md

```diff
@@ -303,11 +303,7 @@ Links in content are black and underlined. They adapt to their context otherwise
 
 ### With chevron
 
-Add `.link-chevron` to bold your link and enhance them with a chevron.
-
-{{< example >}}
-<a class="link-chevron" href="#links">This is a sample link with chevron</a>
-{{< /example >}}
+Use our [link chevron helper]({{< docsref "/helpers/icon-link#link-chevron" >}}) to bold your link and enhance them with a chevron.
```

Comment:
Should we also move the associated code to helpers (instead of `type.scss`) ? Can't we handle this with the new utilities/helpers ?

Reply:
Let's try `helpers/icon-link` that would contain `helpers/chevron-link`.

Reply:
Tried it in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1854/commits/fc89d796f730979c2653dab5ef9dc62bf41321ca. Let me know what you think!